server/api/api/endpoints/alerts.py

```diff
@@ -0,0 +1,203 @@
+    auth_info: mlrun.common.schemas.AuthInfo = Depends(deps.authenticate_request),
+    db_session: Session = Depends(deps.get_db_session),
+) -> list[mlrun.common.schemas.AlertConfig]:
+    await run_in_threadpool(
+        server.api.utils.singletons.project_member.get_project_member().ensure_project,
+        db_session,
+        project,
+        auth_info=auth_info,
+    )
+    await server.api.utils.auth.verifier.AuthVerifier().query_project_permissions(
+        project,
+        mlrun.common.schemas.AuthorizationAction.read,
+        auth_info,
+    )
+
+    return await run_in_threadpool(
+        server.api.crud.Alerts().list_alerts, db_session, project
+    )
+
+
+@router.delete(
+    "/{name}",
+    status_code=HTTPStatus.NO_CONTENT.value,
+)
+async def delete_alert(
+    request: Request,
+    project: str,
+    name: str,
+    auth_info: mlrun.common.schemas.AuthInfo = Depends(deps.authenticate_request),
+    db_session: Session = Depends(deps.get_db_session),
+):
+    await run_in_threadpool(
+        server.api.utils.singletons.project_member.get_project_member().ensure_project,
+        db_session,
+        project,
+        auth_info=auth_info,
+    )
+
+    await server.api.utils.auth.verifier.AuthVerifier().query_project_resource_permissions(
+        mlrun.common.schemas.AuthorizationResourceTypes.alert,
+        project,
+        name,
+        mlrun.common.schemas.AuthorizationAction.delete,
+        auth_info,
+    )
+
+    if (
+        mlrun.mlconf.httpdb.clusterization.role
+        != mlrun.common.schemas.ClusterizationRole.chief
+    ):
+        chief_client = server.api.utils.clients.chief.Client()
+        return await chief_client.delete_alert(
+            project=project, name=name, request=request
+        )
+
+    await run_in_threadpool(
+        server.api.crud.Alerts().delete_alert, db_session, project, name
+    )
+
+
+@router.post("/{name}/reset")
+async def reset_alert(
+    request: Request,
+    project: str,
+    name: str,
+    auth_info: mlrun.common.schemas.AuthInfo = Depends(deps.authenticate_request),
+    db_session: Session = Depends(deps.get_db_session),
+):
+    await run_in_threadpool(
+        server.api.utils.singletons.project_member.get_project_member().ensure_project,
+        db_session,
+        project,
+        auth_info=auth_info,
+    )
+    await server.api.utils.auth.verifier.AuthVerifier().query_project_resource_permissions(
+        mlrun.common.schemas.AuthorizationResourceTypes.alert,
+        project,
+        name,
+        mlrun.common.schemas.AuthorizationAction.update,
+        auth_info,
+    )
+
+    if (
+        mlrun.mlconf.httpdb.clusterization.role
+        != mlrun.common.schemas.ClusterizationRole.chief
+    ):
+        chief_client = server.api.utils.clients.chief.Client()
+        return await chief_client.reset_alert(
+            project=project, name=name, request=request
+        )
+
+    logger.debug(f"Resetting alert {name}", project=project)
```

Comment:
```suggestion
    logger.debug("Resetting alert", project=project, name=name)
```

true for all occurrences